### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by mkeilman**
See commit messages for details

### CODE REVIEW

File: sirepo/package_data/template/warpvnd/default-data.json [hunk 1]
```diff
@@ -30,8 +30,7 @@
         },
         "particle3d": {
             "renderCount": "10",
```
Comment (robnagler) [inline sirepo/package_data/template/warpvnd/default-data.json]: let's try to keep these alphabetized.

File: sirepo/package_data/static/js/sirepo-plotting.js [hunk 2]
```diff
@@ -4300,7 +4305,7 @@ SIREPO.app.directive('particle3d', function(appState, panelState, requestSender,
                     zAxisRight = Math.min(sceneZEnds[1][0], clippedZEnds[1][0]);
                     zAxisBottom = zAxisRight == sceneZEnds[1][0] ? sceneZEnds[1][1] : clippedZEnds[1][1];
                     zAxisProjLen = plotUtilities.dist([zAxisLeft, zAxisTop], [zAxisRight, zAxisBottom]);
-                    var tanTheta = (sceneZEnds[0][1] - sceneZEnds[1][1]) / (sceneZEnds[0][0] - sceneZEnds[1][0]);
+                    tanTheta = (sceneZEnds[0][1] - sceneZEnds[1][1]) / (sceneZEnds[0][0] - sceneZEnds[1][0]);
```
Comment (robnagler) [inline sirepo/package_data/static/js/sirepo-plotting.js]: this seems to be more [impedance mismatches](https://github.com/radiasoft/pykern/wiki/DesignHints#impedance-matches-data-structures-align-with-algorithms) and [differentiate differences](https://github.com/radiasoft/pykern/wiki/DesignHints#differentiate-differences).

File: sirepo/package_data/static/js/sirepo-plotting.js [hunk 3]
```diff
@@ -3595,7 +3596,7 @@ SIREPO.app.directive('particle3d', function(appState, panelState, requestSender,
                 minZSpacing = Math.abs((zmax - zmin)) / numInterPoints;
                 var nearestIndex = 0;
                 indexMaps = [];
-                for(var i = 0; i < zpoints.length; ++i) {
+                for(i = 0; i < zpoints.length; ++i) {
```
Comment (robnagler) [inline sirepo/package_data/static/js/sirepo-plotting.js]: is this zpoints.foreach(z, i)?

File: sirepo/package_data/static/js/sirepo-plotting.js [hunk 4]
```diff
@@ -3595,7 +3596,7 @@ SIREPO.app.directive('particle3d', function(appState, panelState, requestSender,
...
                     var zArr = zpoints[i];  var yArr = ypoints[i];  var xArr = xpoints[i];
                     var l = zArr.length;
```
Comment (robnagler) [inline sirepo/package_data/static/js/sirepo-plotting.js]: this a loop invariant? I'm assuming this is a 3d matrix

File: sirepo/package_data/static/js/sirepo-plotting.js (hunk 2 above)
Comment (mkeilman) [reply sirepo/package_data/static/js/sirepo-plotting.js]: Yes, all that old stuff is still in place while I fix it up...

File: sirepo/package_data/static/js/sirepo-plotting.js (hunk 3 above)
Comment (mkeilman) [reply sirepo/package_data/static/js/sirepo-plotting.js]: I guess so - @moellep said he prefers forEach [but...](https://stackoverflow.com/a/43032526)

**Pull request merged**